packages/rolldown/src/options/input-options.ts

```diff
@@ -137,6 +141,7 @@ export interface InputOptions {
   watch?: WatchOptions | false
   dropLabels?: string[]
   keepNames?: boolean
+  checks?: ChecksOptions
```

Comment:
Should also add it  in schema.

Reply:
Related to #2193 
It seems to have already been updated.